package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "s54_food_places",
+  "version": "1.0.0",
+  "description": "Website where People can explore and help people explore different food places in Punjab.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
```

Comment:
Fill in the author field with the author's name or username. This provides information about who is responsible for the project. [medium]